hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/MapTable.tpl

```diff
@@ -85,13 +85,12 @@
           </table>
         </div>
         <script type="text/javascript">
-          CRM.$(function($) {
-            {if $mappingDetailsError }
-              $('#saveDetails').show();
-            {else}
-              $('#saveDetails').hide();
-            {/if}
-          });
+          {if $mappingDetailsError }
+            cj('#saveDetails').show();
+          {else}
+            cj('#saveDetails').hide();
```

Comment:
Just noticed this now while reviewing your updates. Why changing _$_ to _cj_? Is this something new in 4.7? It looks odd to me for a few reasons:
- Anyone looking at the code would know that $ is a jQuery object. On the other hand, **cj** is kind of meaningless. Are they the same thing?
- A $ variable is still being passed as an argument to the callback, so why not use it?